static/templates/compose_banner/compose_banner.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="compose_banner {{banner_type}} {{classname}}" data-stream-id="{{stream_id}}" data-topic-name="{{topic_name}}">
```

Comment:
The `data-stream-id` and `data-topic-name` fields seem like they should be inside a conditional -- won't this result in something like `data-stream-id="undefined"` for a banner appearing attached to a private message conversation...

Reply:
Got confused about how to do this and asked on [CZO](https://chat.zulip.org/#narrow/stream/6-frontend/topic/conditional.20data.20in.20template/near/1469288)

Reply:
ok done :)

Reply:
Looks great!